Dockerfile

```diff
@@ -343,10 +348,14 @@ USER root
 #
 ################################################################################
 
-ENV HUGO_VERSION 0.85.0
```

Comment:
I was trying to see if they follow semver and feels like they don't(?)
- https://github.com/gohugoio/hugo/releases

I guess there's no easy way of changing this default without uninteded impact but wondering if we could lower this by sharing some update (forum post?) on the matter?

Reply:
Good point will reach out to support

Reply:
Fun fact: The documented version is off with the actual version:
https://github.com/netlify/build-image/blob/focal/included_software.md
https://github.com/netlify/build-image/blob/focal/Dockerfile#L346

Besides that, I'm wondering if we have any data on hugo versions being used?
A simple forum update might work for support to point folks to if any issues.

Reply:
I reverted to the original version to get this PR over the finishing line